tests/upgrade/test_upgrade.py

```
@@ -0,0 +1,88 @@
+        container, master_port = run_db(image)
+
+        # wait for DB to accept connections
+        time.sleep(2)
+
+        master_db = FalkorDB(port=master_port)
+
+        # Select the social graph
+        master_graph = master_db.select_graph("upgrade")
+
+        # Generate random graph
+        nodes, edges = create_random_schema()
```

Ruff

`create_random_schema` may be undefined, or defined from star imports

(F405)